desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -15,11 +15,60 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import json
+
 from django.shortcuts import render
 from django.http import HttpResponse
-from desktop.lib.django_util import login_notrequired
+from desktop.lib.django_util import login_notrequired, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from django.conf import settings
+from slack_sdk import WebClient
+
+LOG = logging.getLogger(__name__)
+
+SLACK_VERIFICATION_TOKEN = getattr(settings, 'SLACK_VERIFICATION_TOKEN', None)
+SLACK_BOT_USER_TOKEN = getattr(settings, 'SLACK_BOT_USER_TOKEN', None)
 
 
+Client = WebClient(token=SLACK_BOT_USER_TOKEN)
+BOT_ID = Client.api_call('auth.test')['user_id']
```

Comment:
Here it seems that we are going to create the Client event if the bot is disabled?

Reply:
Checked if the bot is disabled or not in the channel in the get_bot_id function